src/Analysim.Web/Controllers/AccountController.cs

```diff
@@ -335,8 +335,8 @@ private async Task SendEmailConfirmationEmail([FromForm] AccountRegisterVM user,
 
         /*
          * Type : POST
-         * URL : /api/account/forgotPassword
-         * Description: Create and return new User
+         * URL : /api/account/ForgotPassword
```

Comment:
@kiyros why did you change to uppercase? All other endpoints are lowercase.

Reply:
fixed in recent update, I wrote it as uppercase, then changed it to lowercase to match the code convention